superset/datasets/commands/importers/v0.py

```diff
@@ -316,6 +316,7 @@ def run(self) -> None:
                     )
                     dataset["database_id"] = database.id
                     SqlaTable.import_from_dict(db.session, dataset, sync=self.sync)
+                db.session.commit()
```

Comment:
seems that `if` expression can't commit as well. This change should add more unit test.